man/ctags.1.rst.in

```diff
@@ -1420,12 +1420,16 @@ are not listed here. They are experimental or for debugging purpose.
 	name. [Ignored in etags mode]
 
 ``--tag-relative[=yes|no]``
-	Indicates that the file paths recorded in the tag file should be
+	Specyfing yes indicates that the file paths recorded in the tag file should be
```

Comment:
I would follow the format language in other options if you have a doubt (see --sort for example):

```
The yes value  (the default when unspecified) indicates...
The X value indicates...
etc
```